### PR TITLE
Add french language

### PR DESCRIPTION
**Pull request by Ilithy**

### CODE REVIEW

**Comment by Ilithy** [comment]
Hi,
Normally the translation in French is complete, if necessary, do not hesitate to let me know of modifications to be made, thank you for this excellent application

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <string name="show_switch_lang_button">Afficher le bouton de changement de langue</string>
+    <string name="use_popup_mode">Utiliser le mode pop-up</string>
+    <string name="about_this_app">À propos de cette app</string>
+    <string name="version">Version</string>
+
+    <string name="key_dark_mode" translatable="false">defaultDarkMode</string>
+    <string name="key_switch_lang_button" translatable="false">isEnabledSwapLangButton</string>
+    <string name="key_switch_popup_mode" translatable="false">isEnabledPopupMode</string>
+    <string name="key_version" translatable="false">appVersion</string>
+    <string name="link_github_release" translatable="false">https://github.com/sakusaku3939/DeepLAndroid/releases</string>
```
Comment (fm-sys) [inline app/src/main/res/values-fr/strings.xml:23]: Thanks for your contribution!

Strings which are marked as `translatable="false"` shouldn't be present in the translations file. See https://github.com/sakusaku3939/DeepLAndroid/pull/65#discussion_r782064458 where we had the same discussion...

Comment (Ilithy) [reply app/src/main/res/values-fr/strings.xml]: Sorry and thanks, it's done

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <string name="app_name">DeepL</string>
+    <string name="image">Image</string>
+    <string name="copy_clipboard">Le texte a été copié</string>
+    <string name="network_err">Oups, échec de connexion au serveur de DeepL</string>
+    <string name="title_activity_settings">Paramètres</string>
+    <string name="themeSettings">Paramètres du thème</string>
+    <string name="theme">thème</string>
+    <string name="theme_light">Clair</string>
+    <string name="theme_dark">Obscur</string>
+    <string name="theme_battery">Défini par l'économiseur de batterie</string>
```
Comment (sakusaku3939) [inline app/src/main/res/values-fr/strings.xml:11]: The `'` in `l'économiseur` needs to be escaped.

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <string name="theme_default">Identique au système</string>
+    <string name="translation_screen_options">Options de l'écran de traduction</string>
```
Comment (sakusaku3939) [inline app/src/main/res/values-fr/strings.xml:13]: the same as above

**Review by sakusaku3939 — APPROVED** [review]
Thanks for adding the translations!
I looked into it since the CI didn't pass, and found that the translated text needs to be escaped.

File: app/src/main/res/values-fr/strings.xml (hunk 3 above)
Comment (Ilithy) [reply app/src/main/res/values-fr/strings.xml]: sorry for this omission, it has been corrected, thank you

**Comment by sakusaku3939** [comment]
LGTM
Thank you!

**Pull request merged**